docker-compose.yml

```diff
@@ -124,6 +124,7 @@ services:
       - "10080:10080"
     volumes:
       - ./s3/tidbconf/tidb.toml:/tidb.toml:ro
+      - ./tidb_data:/tmp/tidb

```

Comment:
Is it better to use an absolute path instead of './tidb_data' ?

Reply:
It can also be put in /data or /var/lib, but my concern is that if the user doesn't run multi-cloud with root, it may met some permission issue to create/write file in /data or /var/lib, so maybe it's better to put it in current directory and add tidb_data to .gitignore, to avoid that tidb_data was deleted as untracked file by "git clean".